[2021-07-16 10:01:58,648] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2021-07-16T10:01:41.905293+00:00 [queued]>
[2021-07-16 10:01:59,037] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2021-07-16T10:01:41.905293+00:00 [queued]>
[2021-07-16 10:01:59,037] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 10:01:59,037] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-16 10:01:59,037] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 10:01:59,168] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_2> on 2021-07-16T10:01:41.905293+00:00
[2021-07-16 10:01:59,171] {standard_task_runner.py:52} INFO - Started process 1683 to run task
[2021-07-16 10:01:59,176] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_2', '2021-07-16T10:01:41.905293+00:00', '--job-id', '6', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpsdr7msv5', '--error-file', '/tmp/tmp0u5xv31y']
[2021-07-16 10:01:59,177] {standard_task_runner.py:77} INFO - Job 6: Subtask task_2
[2021-07-16 10:02:00,405] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_2 2021-07-16T10:01:41.905293+00:00 [running]> on host apache-airflow.us-central1-a.c.eminent-it-info.internal
[2021-07-16 10:02:01,970] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2021-07-16T10:01:41.905293+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-16T10:01:41.905293+00:00
[2021-07-16 10:02:01,971] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-16 10:02:01,971] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-07-16 10:02:01,977] {subprocess.py:75} INFO - Output:
[2021-07-16 10:02:04,979] {subprocess.py:83} INFO - Command exited with return code 0
[2021-07-16 10:02:05,837] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_2, execution_date=20210716T100141, start_date=20210716T100158, end_date=20210716T100205
[2021-07-16 10:02:06,618] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-16 10:02:06,642] {local_task_job.py:151} INFO - Task exited with return code 0
